client/src/views/pages/Home.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+
+import PollsList from '../components/PollsList'
+
+export class Home extends React.Component {
+  render() {
+    const { polls } = this.props;
+    return !!polls ? (
+      <div>
+        <PollsList
+          polls={polls}
+          authed={this.props.authed}
+          updatePollVote={this.props.updatePollVote}
+        />
+      </div>
+    ) : (
+      <div>No content.</div>
+    )
+  }
```

Comment:
Could you try adding a `loading` state here?

Reply:
Yeah, for sure! I do have "loading" state in my reducer. Just like I said about the local CORS issue. My progress on the business layer was stucked for a very very long time. I'll improve these within this week.
Thank you so much for the suggestions.